### DISCUSSION TITLE
extension is showing the following problem with my dbt-project.yml but dbt CLI is working fine.

### DISCUSSION THREAD

**Discussion by vmkonda** [AltimateAI/vscode-dbt-power-user#1087] [Q&A]
Error: "dbt Configuration is invalid : release unlocked lock"

I am using vscode + Remote SSH into an AWS ec2 instance + extension

**Comment by anandgupta42** [comment]
Can you run

1. dbt clean
2. dbt deps

Let us know if this fixes the issue.

**Comment by vmkonda** [comment]
Did that. Still the error is there.
________________________________

**Comment by morindo** [comment]
I'm having the same problem.

![image](https://github.com/AltimateAI/vscode-dbt-power-user/assets/1229161/7e5c666e-3afa-45c8-b8ff-da6a54d3cf7a)
